src/pages/app-development/app-submission-guidelines.md

```diff
@@ -83,10 +84,18 @@ The following requirements must be met for your app to be accepted. If your app
 ### Project structure
 
 - Configuration files
-  - **Environment variables**: Provide a clear `.env.dist` file containing all needed keys. Remove any unused keys.
+  - **Environment variables**: Provide a clear `.env.dist` file containing all needed keys used by YAML files. Remove any unused keys.
+  - **Package metadata**: Ensure `package.json` is updated with an app-specific `name`, `version`, and `author`.
   - **YAML configuration**: Review `deploy.yaml` and `app.config.yaml` for accurate app IDs, event configs, and scopes.
   - **Commerce product**: Define `commerce` as a required product in `app.config.yaml`. See [required products](https://developer.adobe.com/commerce/extensibility/app-development/required-products/) for more information.
-  - **Package metadata**: Ensure `package.json` is updated with an app-specific `name`, `version`, and `author`.
+
```

Comment:
@jhadobe the link https://developer.adobe.com/commerce/extensibility/app-development/required-products/ for `required products` is broken, and could you check which link we should use?

Reply:
@kuflower I assume this is some sort of hallucination.